pkgs/misc/drivers/xow/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, libusb, gitMinimal }:
+
+stdenv.mkDerivation rec {
+  pname = "xow";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "medusalix";
+    repo = "xow";
+    rev = "v${version}";
+    sha256 = "03ajal91xi52svzy621aa4jcdf0vj4pqd52kljam0wryrlmcpbr3";
+  };
+
+  makeFlags = "BUILD=RELEASE";
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ gitMinimal ];
+  buildInputs = [ libusb ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp xow $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/medusalix/xow";
+    description = "Linux driver for the Xbox One wireless dongle";
+    license = licenses.gpl2;
```

Looking at the source code:

```suggestion
    license = licenses.gpl2Plus;
```

Unfortunately, GitHub only shows what the attached license file contains, not the actual license.

---

Oh, thanks! Fixed.